CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# How To Contribute
+
```

Comment:
What are the Code of Conduct rules?  I would recommend https://www.contributor-covenant.org/version/2/0/code_of_conduct/